mitmproxy/options.py

```diff
@@ -74,6 +74,13 @@ def __init__(self, **kwargs) -> None:
             None,
             "Set supported ciphers for mitmproxy <-> server connections using OpenSSL syntax.",
         )
+        self.add_option(
+            "curve",
+            Optional[str],
+            None,
+            "Specify an elliptic curve for ECDHE key exchange between mitmproxy <-> server connections using OpenSSL "
+            "syntax."
```

Comment:
Add a curve example here?

Reply:
I added 3 example curve, however I was not sure whether I should specify the command `openssl ecparam -list_curve`. This command lists the supported curves, at least for Linux.